Website/src/app/layout.tsx

```diff
@@ -1,10 +1,18 @@
-import RootLayout from "@/components/layout"
+import Layout from "@/app/components/layout_base"
+import {inter, meta_info} from "@/lib/common";
 
-export const metadata = {
-  title: 'Create Next App',
-  description: 'Generated by create next app',
-}
+export const metadata = meta_info;
 
-export default RootLayout;
+export default function RootLayout({children,}: { children: React.ReactNode }) {
```

Comment:
Naming here seems inconsistent: the `layout_base` file contains the`Layout` and the `layout` file the `RootLayout`.

Reply:
Files named `layout.ts(x)` (in the app-directory) have special meaning in next.js as these define a component that "wrapps" all page components in subdirectories. As I however need to use the same layout in both the app-, as well as the pages-directory. I factored the relevant parts into a component.

However I agree that naming things is hard